src/Image/Cache.php

```diff
@@ -168,7 +168,7 @@ function ($parser, $name, $attributes) use ($options, $parsed_url, $full_url) {
                             }
                         }
                     },
-                    false
+                    function () {}
```

Comment:
Not sure why we used false previously. Wouldn't it be better to just specify `null` here?

Reply:
Sure, don't know why I made it so complicated. Too many things to do at the same time.